src/components/Articles/ResolutionDates.vue

```diff
@@ -1,90 +1,95 @@
 <template>
   <div id="resolution-dates">
-    <template>
-
-      <!-- Base Display -->
-      <v-row no-gutters class="mb-4">
-        <v-col cols="3">
-          <label>
-            <span>Resolution or <br> Court Order Dates</span>
-          </label>
-        </v-col>
-
-        <v-col cols="7">
-          <span class="info-text">Indicate the date of the resolution or court order used to alter the company's share
-            structure or the special rights and restrictions of a class or series of shares:</span>
-        </v-col>
-
-        <v-col cols="2" class="align-right">
-          <v-btn v-if="!isEditing"
-                 id="add-resolution-date"
-                 class="add-btn"
-                 text color="primary"
-                 @click="isEditing = true"
+
```

Comment:
Please "hide whitespace changes" in the options menu above to better see these changes.